app/views/device_stories/_search_form.html.erb

```diff
@@ -1,5 +1,5 @@
-<%= form_tag(device_stories_path, remote: true, :method => "get", enforce_utf8: false,
-             class: 'navbar-form navbar-left') do %>
+<%= form_tag(device_stories_path, remote: true, method: :get, enforce_utf8: false,
+             class: 'navbar-form navbar-left', data: { type: 'script' }) do %>
```

Comment:
Is this data tag special some how? Curious how this works.

Reply:
Yes.  `data: { type: 'script' }` will be rendered as `data-type="script"` in HTML. jquery-ujs [reads](https://github.com/rails/jquery-ujs/blob/master/src/rails.js#L116) this value and [pass](https://github.com/rails/jquery-ujs/blob/master/src/rails.js#L147) it to `jQuery.ajax` method option as `dataType`.

For `dataType` in `jQuery.ajax`, please refer [this document](https://api.jquery.com/jquery.ajax/).